### DISCUSSION TITLE
CoKreat release-5.2.0 fresh installation and upgrade documentation

### DISCUSSION THREAD

**Discussion by gandham-santhosh** [Sunbird-coKreat/Community#80] [General Discussions]
Hi Team,
Can you please let us know if the CoKreat fresh installation and upgrade documentation for release-5.2.0 is updated? If not can you please update and share the link

**Comment by mmanimegalai** [comment]
The current installation document has very basic details and needs to be updated. We have already planned this activity as part of 6.0.0. We would need support from DevOps team to get this done 
cc @vinukumar-vs @vaivk369

**Reply by gandham-santhosh** [reply]
@mmanimegalai please let me know what support is required from DevOps?

**Reply by gandham-santhosh** [reply]
cc: @Krishnaj20 @vinukumar-vs

**Comment by mmanimegalai** [comment]
@gandham-santhosh 
We discussed about the need and the support required from your team. Here are the tickets for the same. 
https://project-sunbird.atlassian.net/browse/CO-449
https://project-sunbird.atlassian.net/browse/CO-450

cc @vinukumar-vs
